src/main-refactor/network/10-vpc.tf

```
@@ -0,0 +1,89 @@
+locals {
+  eks_workload_cidrs      = ["10.0.0.0/21", "10.0.8.0/21", "10.0.16.0/21"]
+  eks_control_plane_cidrs = ["10.0.24.0/24", "10.0.25.0/24", "10.0.26.0/24"]
+  probing_lb_cidrs        = ["10.0.27.0/24", "10.0.28.0/24", "10.0.29.0/24"]
+  aurora_cidrs            = ["10.0.30.0/24", "10.0.31.0/24", "10.0.32.0/24"]
+  egress_cidrs            = ["10.0.33.0/24", "10.0.34.0/24", "10.0.35.0/24"]
+  vpce_cidrs              = ["10.0.36.0/24", "10.0.37.0/24", "10.0.38.0/24"]
+  vpn_cidrs               = ["10.0.39.0/24", "10.0.40.0/24", "10.0.41.0/24"]
+  msk_cidrs               = ["10.0.42.0/24", "10.0.43.0/24", "10.0.44.0/24"]
+
+  eks_workload_subnets_names = [for idx, subn in local.eks_workload_cidrs :
+  format("%s-eks-workload-%d-%s", local.project, idx + 1, var.env)]
+
+  eks_control_plane_subnets_names = [for idx, subn in local.eks_control_plane_cidrs :
+  format("%s-eks-cp-%d-%s", local.project, idx + 1, var.env)]
+
+  probing_lb_subnets_names = [for idx, subn in local.probing_lb_cidrs :
```

```suggestion
  int_lbs_subnets_names = [for idx, subn in local.probing_lb_cidrs :
```
Since we need to use APIGW, these load balancer will be internal